src/test/java/org/opensearch/repository/encrypted/security/CryptoIOTests.java

```diff
@@ -5,20 +5,26 @@
 
 package org.opensearch.repository.encrypted.security;
 
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
 import org.opensearch.test.OpenSearchTestCase;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.security.Security;
 
 public class CryptoIOTests extends OpenSearchTestCase {
 
+    static {
```

Comment:
May be use standard JUnit idioms: @BeforeClass?

Reply:
fixed